cmd/config/set.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/datreeio/datree/bl/messager"
+	"github.com/spf13/cobra"
+)
+
+func NewSetCommand(ctx *ConfigCommandContext) *cobra.Command {
+	setCommand := &cobra.Command{
```

Comment:
I think `config set` command should be restricted only for setting `token` for now

Reply:
this is intentional. config set will be able to change any key-value in config.yaml file